### PR TITLE
[ADD] l10n_id : Added new localization for Indonesia

### PR DESCRIPTION
**Pull request by hsh-odoo**
Description of the issue/feature this PR addresses:
 - Task : https://www.odoo.com/web#id=42243&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
 - Pad : https://pad.odoo.com/p/r.0cc60a058652f804d7198ca71209f126

Current behavior before PR:
 - Create a l10n_id localization

Desired behavior after PR is merged:
1. Create a CoA 
2. Taxes

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_id/data/res.country.state.csv [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"id","country_id:id","code","name"
```
Comment (jco-odoo) [inline addons/l10n_id/data/res.country.state.csv]: If we have this in base, why do we need it here?  @jao-odoo

File: addons/l10n_id/data/account.account.tag.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+id,name,applicability
```
Comment (jco-odoo) [inline addons/l10n_id/data/account.account.tag.csv]: where are the tags used?

File: addons/l10n_id/data/res.country.state.csv (hunk 1 above)
Comment (rgo-odoo) [reply addons/l10n_id/data/res.country.state.csv]: @jco-odoo  it was removed in commit dbe454f1faa648dbbe4575e4af2987c4ca1d5510 but looks like bad force push 1ad947b305ec5b7452fb72430b6efa456e8adbad reintroduced this file.

File: addons/l10n_id/data/account.account.tag.csv (hunk 2 above)
Comment (rgo-odoo) [reply addons/l10n_id/data/account.account.tag.csv]: in tax templates

File: addons/l10n_id/data/res.country.state.csv (hunk 1 above)
Comment (hsh-odoo) [reply addons/l10n_id/data/res.country.state.csv]: @rgo-odoo  @jco-odoo  Done

File: addons/l10n_id/data/account.account.tag.csv (hunk 2 above)
Comment (jco-odoo) [reply addons/l10n_id/data/account.account.tag.csv]: Ok, but they are not used in any report... (do they correspond to a vat return report?)

**Comment by hsh-odoo** [comment]
Replaced by #32704

**Pull request closed without merging**